packages/terra-list/tests/jest/List.test.jsx

```diff
@@ -64,6 +64,35 @@ it('should render with thin padded items', () => {
   expect(shallowComponent).toMatchSnapshot();
 });
 
```

Comment:
Do `ariaDetails` need unit tests?

Reply:
You're right, it slipped my mind. I'll add them.

Reply:
[b5fe50d](https://github.com/cerner/terra-core/pull/3688/commits/b5fe50d701cf0421e92b7619003598f1f84b537e)